src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -249,11 +255,20 @@ public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceNa
             }
             catch (HttpRequestException)
             {
-                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider",
-                    $"message:Observer SQL query request failed, query:{query}, statusCode:{response.StatusCode}, response:{result}");
+                var logMessage = $"message:Observer SQL query request failed, route:{route}, query:{query}, " +
+                    $"statusCode:{response.StatusCode}, response:{result}, latencyMs:{timer.ElapsedMilliseconds}";
+                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider", logMessage);
                 throw;
             }
 
+            if (timer.ElapsedMilliseconds > 1000)
+            {
+                var logMessage = $"message:Observer SQL query ran for longer than one second, route:{route}, " +
```

Comment:
You don't need to log this. We can already track this info with existing kusto data from event id `3002`. Which is logged in the `DataProviderLogDecorator` class